python/pyspark/ml/classification.py

```diff
@@ -562,6 +564,11 @@ def evaluate(self, dataset):
         java_blr_summary = self._call_java("evaluate", dataset)
         return BinaryLogisticRegressionSummary(java_blr_summary)
 
+    def __repr__(self):
```

Comment:
So my question here is why we aren't calling the Java/Scala toString method directly as we do in the mllib one and in many of the other models in `regression.py` for the ml one?